.github/workflows/publish.yaml

```diff
@@ -0,0 +1,62 @@
+name: 🚀 Publish assets to NPM
+
+on:
+    release:
+        types: [ published ]
+
+jobs:
+    publish-prime:
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: ./src/Prime/Resources
+        permissions:
+            contents: read
+            id-token: write
+        steps:
+            - uses: actions/checkout@v4
+
+            # Setup .npmrc file to publish to npm
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: '20.x'
+                  registry-url: 'https://registry.npmjs.org'
+
+            - run: npm ci
+            - run: npm publish --provenance --access=restricted
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+    publish-plugin:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+            id-token: write
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - package: "@flasher/flasher-noty"
+                      path: ./src/Noty/Prime/Resources
+                    - package: "@flasher/flasher-notyf"
+                      path: ./src/Notyf/Prime/Resources
+                    - package: "@flasher/flasher-sweetalert"
+                      path: ./src/SweetAlert/Prime/Resources
+                    - package: "@flasher/flasher-toastr"
+                      path: ./src/Toastr/Prime/Resources
+        steps:
+            - uses: actions/checkout@v4
+
+            # Setup .npmrc file to publish to npm
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: '20.x'
+                  registry-url: 'https://registry.npmjs.org'
+
+            - run: npm ci
+              working-directory: ${{ matrix.path }}
+
+            - run: npm publish --provenance --access=restricted
```

Comment:
I'm not really versed in the whole NPM build/publishing, but for now I added `--access=restricted` in case I want to try to publish the package under alternative name but don't want it public (so I think this would be private?)